test/src/main/java/org/apache/accumulo/test/ScanConsistencyIT.java

```diff
@@ -170,34 +173,34 @@ private static void runTest(AccumuloClient client, FileSystem fileSystem, String
       var tableOpsTask = executor.submit(new TableOpsTask(testContext));
 
       // let the concurrent mayhem run for a bit
-      Thread.sleep(60000);
+      Thread.sleep(sleepTime);
 
       // let the threads know to exit
       testContext.keepRunning.set(false);
 
       for (Future<WriteStats> writeTask : writeTasks) {
         var stats = writeTask.get();
-        logMessage(String.format("Wrote:%,d Bulk imported:%,d Deleted:%,d Bulk deleted:%,d",
+        log.info(String.format("Wrote:%,d Bulk imported:%,d Deleted:%,d Bulk deleted:%,d",
```

Comment:
These could use the logging placeholder `{ }` instead of String.format. It does not look like there is any specific conversions or padding.

Reply:
`%,d` (note the comma) converts the integer to be formatted with commas. I don't think `{}` can be used to format the string like this but I'm not sure